terratest/src/test/backingservices/HelmChartParser.go

```diff
@@ -0,0 +1,54 @@
+package backingservices
+
+import (
+	"github.com/gruntwork-io/terratest/modules/helm"
+	"strings"
+	"testing"
+)
+
+type HelmChartParser struct {
+	T              *testing.T
+	SlicedResource []string
+}
+
+func NewHelmConfigParser(helmTest *HelmTest) *HelmChartParser {
+	parsedChart := helm.RenderTemplate(helmTest.T, helmTest.HelmOptions, helmTest.ChartPath, "srs",helmTest.Templates)
+	slicedResource := strings.Split(parsedChart, "---")
+	return &HelmChartParser{T: helmTest.T, SlicedResource: slicedResource}
+}
+
+func (p *HelmChartParser) getResourceYAML(searchOptions SearchResourceOption, resource interface{}) {
+	var d DeploymentMetadata
+	for _, slice := range p.SlicedResource {
+		helm.UnmarshalK8SYamlE(p.T, slice, &d)
+		if (searchOptions.Kind != "" && searchOptions.Kind == d.Kind) && (searchOptions.Name != "" && searchOptions.Name == d.Name) {
+			helm.UnmarshalK8SYaml(p.T, slice, &resource)
+			return
+		}
+	}
+	p.T.Log("Resource not found.", searchOptions)
+	p.T.FailNow()
+}
+
+func (p *HelmChartParser) Find(searchOptions SearchResourceOption, resource interface{}) {
+	var d DeploymentMetadata
+	for _, slice := range p.SlicedResource {
+		helm.UnmarshalK8SYaml(p.T, slice, &d)
+		if (searchOptions.Kind != "" && searchOptions.Kind == d.Kind) && (searchOptions.Name != "" && searchOptions.Name == d.Name) {
+			helm.UnmarshalK8SYaml(p.T, slice, &resource)
+			return
+		}
+	}
+	p.T.Log("Resource not found.", searchOptions)
+	p.T.FailNow()
+}
+func (p *HelmChartParser) Contains(searchOptions SearchResourceOption) bool {
+	var d DeploymentMetadata
+	for _, slice := range p.SlicedResource {
+		helm.UnmarshalK8SYamlE(p.T, slice, &d)
```

Comment:
Using UnmarshalK8SYamlE is like skipping any errors while unmarshalling yaml. Please use UnmarshalK8SYaml instead.

Reply:
That looks like a local setup issue. I believe you will not see these errors in actual travis builds. You can validate that in your account's travis.

Reply:
I have changed it to UnmarshalK8SYaml , but I dont know how to validate in my account's travis